### ISSUE TITLE
boa v0.4

### ISSUE THREAD

**Issue by lllwvlvwlll** [CityOfZion/neo3-boa#113]
Release discussion for boa v0.0.4

**Comment by ixje** [comment]
https://github.com/neo-project/neo/pull/1973/

**Comment by lock9** [comment]
I'll close this so we can focus on v.0.5. @ixje the PR you mentioned is not merged yet 🤔 If it is merged, I think we should create a new issue to adapt neo-boa. I'll close this for now, we can reopen it if you disagree. Thanks!

**Comment by ixje** [comment]
I mentioned the PR such that we cannot forget to update once it is merged

**Comment by lock9** [comment]
Hi @ixje, I created an issue to ensure we don't forget about it.
